JeMPI_Apps/JeMPI_Linker/pom.xml

```diff
@@ -134,6 +134,32 @@
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-slf4j-impl</artifactId>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter -->
+        <dependency>
```

Comment:
@mahaomolise92 should these dependency not be added to the main pom and versions referenced here?

Reply:
@MatthewErispe I have been investigating if it is possible to add this dependencies in the parent POM since we are likely going to need them in all other modules, I will continue to investigate this